PacketDotNet/NullPacket.cs

```diff
@@ -23,10 +23,10 @@
 using System.Threading;
 using PacketDotNet.Utils;
 using PacketDotNet.Utils.Converters;
```

Comment:
Is the format around the #if what you intended? No space before the #if but a space before the #endif?

Reply:
> 
> 
> Btw github can’t display the full changeset here because of its size.

Correct, GitHub's desktop client works OK. The format of the '#endif' was changed by last commit by ReSharper, I still have to undo it, I'll commit it in a few mins.

Reply:
Should be fixed now. It's because ReSharper doesn't cope well with regions and preprocessor directives during formatting, it only checks what's currently used and forces these on top in using statements, rather that respecting the preproc directives.